sdk/liftrqumulo/Azure.ResourceManager.Qumulo/tests/LiftrQumuloManagementTestBase.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.TestFramework;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.LiftrQumulo.Tests
```

Comment:
I would do a search on the entire solution and replace all occurrences of `LiftrQumulo` with `Qumulo` since there are still more cases of this.

Reply:
done